.gitignore

```diff
@@ -134,12 +134,7 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-# Developers
```

Comment:
I don't think these lines should be removed

Reply:
I added them back